includes/validate.php

```diff
@@ -465,10 +470,40 @@ function edac_show_draft_posts( $query ) {
 	}
 
 	// If the passed token is no longer valid, we do nothing and return early.
-	if ( false === edac_is_valid_nonce( 'draft-or-pending-status', $url_token ) ) {
+	if ( ! wp_verify_nonce( $url_token, EDAC_VALIDATE_NONCE_ACTION ) ) {
 		return;
 	}
 
 	// If we've reached this point, alter the query to include 'publish', 'draft', and 'pending' posts.
 	$query->set( 'post_status', array( 'publish', 'draft', 'pending' ) );
 }
+
+/**
+ * Get current user cookies
+ *
+ * @return string
+ */
+function edac_get_current_user_cookies(): string {
+	$cookies = array();
+	// phpcs:ignore WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE -- User must be logged in to trigger this so it's a valid use for cookies on the server.
+	foreach ( $_COOKIE as $key => $value ) {
+		$cookies[] = $key . '=' . $value;
+	}
```

Comment:
This might be why not doing this before was considered —VIP rules flag that cookies may not be reliable due to caching. I don't think that's a concern, though; the user can trigger this from a logged-in state only, where the session cookie must exist and be transferred on requests.